### PR TITLE
Docs: 8.4 merge updates for Reference Pages

### PR DESCRIPTION
**Pull request by dyozie**
Covers changes to:

ALTER DATABASE
ALTER SERVER
ALTER TABLE
ALTER USER MAPPING
CLUSTER
CLUSTERDB
COPY
CREATE DATABASE

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/sql_commands/ALTER_USER_MAPPING.xml [hunk 1]
```diff
@@ -7,26 +7,26 @@
...
-    SERVER <varname>server_name</varname>
+            <codeblock id="sql_command_synopsis">ALTER USER MAPPING FOR { <varname>username</varname> | USER | CURRENT_USER | PUBLIC }
+    SERVER <varname>servername</varname>
     OPTIONS ( [ ADD | SET | DROP ] <varname>option</varname> ['<varname>value</varname>'] [, ... ] )</codeblock>
         </section>
         <section id="section3">
             <title>Description</title>
-            <p><codeph>ALTER USER MAPPING</codeph> changes the definition of a user mapping for a foreign server.</p>
-            <p>The owner of a foreign server can alter user mappings for that server for any user. Also, a user granted <codeph>USAGE</codeph> privilege on the server can alter a user mapping for their own user name.</p>
+            <p><codeph>ALTER USER MAPPING</codeph> changes the definition of a user mapping for a
```
Comment (lisakowen) [inline gpdb-doc/dita/ref_guide/sql_commands/ALTER_USER_MAPPING.xml]: i updated this page to 9.4 in a previous PR.  i think we should keep the original text.

File: gpdb-doc/dita/utility_guide/client_utilities/clusterdb.xml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE topic
   PUBLIC "-//OASIS//DTD DITA Composite//EN" "ditabase.dtd">
-<topic id="topic1"><title id="oy139586">clusterdb</title><body><p id="client_util_desc">Reclusters tables that were previously clustered with <codeph>CLUSTER</codeph>.</p><section id="section2"><title>Synopsis</title><codeblock id="client_util_synopsis"><b>clusterdb</b> [<varname>connection-option</varname> ...] [<b>-v</b>] [<b>-t</b> <varname>table</varname>] [[<b>-d</b>] <varname>dbname</varname>]
+<topic id="topic1"><title id="oy139586">clusterdb</title><body><p id="client_util_desc">Reclusters tables that were previously clustered with <codeph>CLUSTER</codeph>.</p><section id="section2"><title>Synopsis</title><codeblock id="client_util_synopsis"><b>clusterdb</b> [<varname>connection-option</varname> ...] [<b>--verbose | -v</b>] [<b>--table | -t</b> <varname>table</varname>] [[<b>--dbname | -d</b>] <varname>dbname</varname>]
```
Comment (lisakowen) [inline gpdb-doc/dita/utility_guide/client_utilities/clusterdb.xml]: interesting that the --long-form is displayed first (i.e. --dbname | -d).  don't most of the docs display the short/single letter form first?  looking farther down to the Options section, the single letter version is displayed first.

File: gpdb-doc/dita/ref_guide/sql_commands/ALTER_USER_MAPPING.xml (hunk 1 above)
Comment (dyozie) [reply gpdb-doc/dita/ref_guide/sql_commands/ALTER_USER_MAPPING.xml]: Whoops - didn't see your request until after the merge.  I'll change this text back to match 9.4.

File: gpdb-doc/dita/utility_guide/client_utilities/clusterdb.xml (hunk 2 above)
Comment (dyozie) [reply gpdb-doc/dita/utility_guide/client_utilities/clusterdb.xml]: I did find this strange, but opted to keep the file in line with the postgresql source for now.

**Pull request merged**